bootstrapper-maven-plugin/src/main/resources/templates/Spec.java

```diff
@@ -0,0 +1,15 @@
+package {{groupId}};
+
+public class {{artifactClassId}}Spec {
+
+    private String value;
+
+    public String getValue() {
+        return value;
+    }
+
+    public {{artifactClassId}}Spec setValue(String value) {
```

Comment:
I also don't agree with this `return this` idiom in setters. While it makes sense for builder-like classes, it makes less sense for "value" classes and might confuse tools that rely on Java beans conventions (such as the CRD generator)

Reply:
agree, changed.